ci-jitpack-properties.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
+set +uex
+
+echo "" >> ~/.gradle/gradle.properties
+echo "odysseyRepositoryUsername=${JITPACK_AUTH_TOKEN}" >> ~/.gradle/gradle.properties
+echo "odysseyRepositoryPassword=" >> ~/.gradle/gradle.properties
```

Comment:
The empty password is what the `PasswordCredentials` is expecting I guess right? Or is this leftover from something we no longer need?